front/pages/api/v1/w/[wId]/assistant/agent_configurations/[name].ts

```diff
@@ -0,0 +1,134 @@
+import type { WithAPIErrorResponse } from "@dust-tt/types";
```

Comment:
This endpoint should probably be renamed to `/search?query=<my_query>`

Reply:
Moved